tests/opensource-suites/collections/contentTableItemsLinks.test.ts

```diff
@@ -0,0 +1,28 @@
+import {openTestPage} from '../../utils';
+import datalensTest from '../../utils/playwright/globalTestDefinition';
+import {CollectionsPage} from '../../page-objects/collections';
+import {WorkbooksUrls, CollectionsUrls} from '../../constants/constants';
+
+datalensTest.describe('Collections page', () => {
+    datalensTest.describe('Content table', () => {
+        datalensTest.beforeEach(async ({page}) => {
+            await openTestPage(page, `/collections`);
+        });
+
+        datalensTest('Workbook row link leads to workbook page', async ({page}) => {
+            const collectionsPage = new CollectionsPage({page});
+
+            await collectionsPage.clickWorkbookRowLink();
+
+            await page.waitForURL(WorkbooksUrls.E2EWorkbook);
+        });
+
+        datalensTest('Collection row link leads to collection page', async ({page}) => {
+            const collectionsPage = new CollectionsPage({page});
+
+            await collectionsPage.clickCollectionRowLink();
+
+            await page.waitForURL(CollectionsUrls.E2ECollection);
```

Comment:
If the purpose of the test was to check the formation of correct links, then it is probably fine. 
But if it was necessary to check the transition to the page, then it is not enough to check the url - you need to check some other sign that the expected page has been rendered.

Reply:
added some pages content visibility checks